src/NotificationsApp.vue

```diff
@@ -11,7 +11,7 @@
 				class="notifications-button__icon"
 				:size="20"
 				:title="t('notifications', 'Notifications')"
-				fill-color="var(--color-primary-text)" />
+				fill="currentColor" />
```

Comment:
Here it should be also removed

Reply:
That is the "backport" in https://github.com/nextcloud/notifications/pull/1534#issuecomment-1519769359